todo-list.js

```diff
@@ -0,0 +1,60 @@
+let taskList = []
+// const taskName = []
+// let taskCounter = 0;
+class Task  {
+    constructor(name, dueDate, isDone) {
+        this.taskId = Date.now();
+        this.name= name;
+        this.dueDate=dueDate;
+        this.isDone = isDone;
```

Comment:
This is not being modified or used anywhere else. You can call a function in the checkbox and toggle this value to true/false depending on the checkbox state